public/app/plugins/datasource/grafana/datasource.ts

```diff
@@ -202,6 +226,7 @@ export class GrafanaDatasource extends DataSourceWithBackend<GrafanaQuery> {
       if (!Array.isArray(target.tags) || target.tags.length === 0) {
         return Promise.resolve({ data: [] });
       }
+      const templateSrv = getTemplateSrv();
```

Comment:
^^ necessary?

Reply:
seems to be used for tags..it was just moved a few lines down.